pibooth/plugins/picture_plugin.py

```diff
@@ -127,6 +127,10 @@ def state_processing_do(self, cfg, app):
                                                                           factory=default_factory)
                     self.factory_pool.add(factory)
 
+    @pibooth.hookimpl
+    def state_processing_exit(self, app):
+        app.count.taken += 1  # Do it here because 'print' state can be skipped
```

Comment:
I would have done it at the exit of the processing state

Reply:
This is the case, no? Or would you mean that the comment is useless?

Reply:
I misread, I was seeing state_finished_exit...